thorlcr/master/thmastermain.cpp

```diff
@@ -780,6 +782,22 @@ int main( int argc, char *argv[]  )
 
         if (registry->connect())
         {
+            if (channelsPerSlave > 1)
+            {
+                for (unsigned s=0; s<slavesPerNode; s++)
+                {
+                    StringBuffer slaveStr;
+                    for (unsigned c=0; c<channelsPerSlave; c++)
+                    {
+                        unsigned o = s + (c * slavesPerNode);
```

Comment:
Unless you have 1 node defined in your cluster, this will be incorrect.
Should be : 
```
unsigned o = s + (c * queryNodeClusterWidth());
```